### DISCUSSION TITLE
Column type SET

### DISCUSSION THREAD

**Discussion by Ahmadh26** [prisma/prisma#14357] [Q&A]
let's say i have the following: 

```prisma 
model Packaging {
id                  Int                   @id @default(autoincrement())
type packageType
// ......Rest of columns here
}

enum packageType {
  plastic
  cardboard
  stainless
  fibre
  carton
}
```

how can i make the column type as `SET` and not ENUM as i want to choose multiple values .. is it possible?

i have tried doing `type packageType[]` but it errored .. 

Thanks.

**Comment by well-balanced** [comment]
I think you are looking for this document guide, isn't it?

https://www.prisma.io/dataguide/mysql/introduction-to-data-types#enumerated-and-set-types

**Reply by Ahmadh26** [reply]
it is, but i how can i implement it in my schema? that's what i cannot figure out

**Comment by austincrim** [comment]
Hey @Ahmadh26 👋,

`SET` is not supported by Prisma yet. Please take a look at [this outstanding feature request](https://github.com/prisma/prisma/issues/10620) and leave your +1 and specific use case.

In the meantime, you could [customize a generated Prisma migration](https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate/include-unsupported-database-features#customize-a-migration-to-include-an-unsupported-feature) to add the `SET` column, but you will only be able to interact with it through [`$queryRaw`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#queryraw).

**Reply by Ahmadh26** [reply]
Hey @austincrim 
Where can I find the feature request? :)

**Reply by austincrim** [reply]
Sorry, forgot the link!

https://github.com/prisma/prisma/issues/10620

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
